src/viewer/overlay/index.tsx

```diff
@@ -11,6 +11,9 @@ import {
 } from "../../utils/twitch";
 import Root from "./Root";
 
+ga.create("UA-81509238-9");
+ga.getAll().forEach(tracker => tracker.set("anonymizeIp", true));
```

Comment:
Also here, although we still need to track the page view, so let's add Google's official recommendation there in the second line:

```suggestion
ga("set", "anonymizeIp", true);
ga("send", "pageview");
```